src/main/resources/fabric.mod.json

```diff
@@ -29,7 +29,7 @@
   "depends": {
     "fabricloader": ">=0.14.9",
     "fabric": "*",
-    "minecraft": "1.19.*",
+    "minecraft": ">=1.20-",
```

Comment:
I haven't seen this yet, what does the minus in the version number mean?

Reply:
the - supports any version after 1.20, as in 1.20.1, and 1.20-rc1 etc

Reply:
Cool!